lib/OpenQA/WebAPI/Controller/API/V1/Comment.pm

```diff
@@ -147,6 +147,52 @@ sub create ($self) {
 
 =over 4
 
+=item create_many()
+
+Adds new comments to the specified jobs. Returns a 200 code with a JSON containing the
+new comment ids or 400 if no text is specified for the comment.
+
+=back
+
+=cut
+
+sub create_many ($self) {
+    my $validation = $self->validation;
+    $validation->required('text')->like(qr/^(?!\s*$).+/);
+    $validation->required('job_id')->num(0, undef);
```

Comment:
Please add the parameters to the documentation. In some places through the API, we use jobid, in others job_id, testid, etc.

Same for the delete_many. (A ticket would also suffice)